lib/controllers/ServiceFabrikAdminController.js

```diff
@@ -605,6 +607,40 @@ class ServiceFabrikAdminController extends BaseController {
         .send({}));
   }
 
+  // Method for backup reporting
+  serviceInstanceBackupSummary(req, res) {
+    if (!req.params.instance_id || !req.query.start_time || !req.query.end_time) {
+      throw new BadRequest('instance_id | start_time | end_time required');
+    }
+    if (!moment(req.query.start_time, 'YYYY-MM-DD', true).isValid() ||
```

Comment:
please move this to some constants file - 'YYYY-MM-DD' and then reuse the constant everywhere.